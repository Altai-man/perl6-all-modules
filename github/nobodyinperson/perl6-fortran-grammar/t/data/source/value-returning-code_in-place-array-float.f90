(/ 1.34, 2.45, 3.45 /)
