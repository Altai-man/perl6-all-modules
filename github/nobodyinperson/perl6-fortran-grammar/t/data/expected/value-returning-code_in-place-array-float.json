{
  "in-place": {
    "in-place-array": {
      "numbers": {
        "number": [
          {
            "float": {
              "digits": [
                "1",
                "34"
              ]
            }
          },
          {
            "float": {
              "digits": [
                "2",
                "45"
              ]
            }
          },
          {
            "float": {
              "digits": [
                "3",
                "45"
              ]
            }
          }
        ]
      }
    }
  }
}
